### PR TITLE
Lesson 1. Tasks 1-5.

### PR DESCRIPTION
**Pull request by VikaTamazian**
Lesson 1 Tasks 1-5

### CODE REVIEW

File: Lesson1/src/com/task4/Solution2.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ * Определить число, полученное выписыванием в обратном порядке цифр
+ * любого 4-х значного натурального числа n.
+ * Решение второе.
+ */
+public class Solution2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введите четыре цифры вашего числа: ");
```
Comment (TheFallen058) [inline Lesson1/src/com/task4/Solution2.java:13]: Not clear that I should you space between numbers)
if you just enter 4567 - application looks freezed

File: Lesson1/src/com/task4/Solution3.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    public static void printArray(String[] values) {
+        for (String value : values) {
+            System.out.println(value);
+        }
+    }
+
+    public static void reverseString(String[] str) {
+        for (int i = 0; i < str.length; i++) {
```
Comment (TheFallen058) [inline Lesson1/src/com/task4/Solution3.java:31]: 'for' loop replaceable with enhanced 'for' 
please pay attention to IDE hints

File: Lesson1/src/com/task5/Solution2.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.Set;
+
+/**
+ * Дано любое натуральное 4-х значное число.
+ * Верно ли, что все цифры числа различны?
+ */
+
+public class Solution2 {
+    public static void main(String[] args) {
+        Set<Integer> list = new HashSet<>();
```
Comment (TheFallen058) [inline Lesson1/src/com/task5/Solution2.java:13]: Bad style to name Set variable as list, brings a lot of misunderstanding

**Review by TheFallen058 — APPROVED** [review]
PLease check my comments

**Comment by VikaTamazian** [comment]
Lesson1

**Pull request merged**